### PR TITLE
feat: Retiring non-AVM modules

### PR DESCRIPTION
**Pull request by matebarabas**
## Description

This PR retires the remaining 17 non-AVM modules. Apart from some minor language updates, such as clearly showing in the `README.md`'s header whether a module is retired or replaced, it also includes updates the `ARCHIVED.md` file of the already retired modules with direct links to their source code.

The following modules will be retired:

- `app/dapr-containerapp`
- `app/dapr-containerapps-environment`
- `azure-gaming/game-dev-vm`
- `azure-gaming/game-dev-vmss`
- `compute/custom-image-vmss`
- `cost/resourcegroup-scheduled-action`
- `cost/subscription-scheduled-action`
- `deployment-scripts/aks-run-command`
- `deployment-scripts/aks-run-helm`
- `deployment-scripts/build-acr`
- `deployment-scripts/create-kv-certificate`
- `deployment-scripts/create-kv-sshkeypair`
- `deployment-scripts/import-acr`
- `deployment-scripts/wait`
- `lz/sub-vending`
- `observability/grafana`
- `storage/data-explorer`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
 
-To provide our customers with a unified experience, Azure Verified Modules ([AVM](https://aka.ms/AVM)) will become the single Microsoft standard for Bicep modules, published to the Public Bicep Registry, via this repository.
+To provide our customers with a unified experience, Azure Verified Modules ([AVM](https://aka.ms/AVM)) is now the single Microsoft standard for Bicep modules, published to the Public Bicep Registry, via this repository.
 
-Going forward, new modules will need to be developed and published in accordance with the [AVM specifications](https://azure.github.io/Azure-Verified-Modules/specs/module-specs/). Module proposals for new, non-AVM modules will no longer be accepted. To propose a new AVM module, you can file a Module Proposal, [here](https://aka.ms/AVM/ModuleProposal).
+Going forward, new modules need to be developed and published in accordance with the [AVM specifications](https://azure.github.io/Azure-Verified-Modules/specs/module-specs/). Module proposals for new, non-AVM modules are no longer accepted. To propose a new AVM module, you can file a Module Proposal, [here](https://aka.ms/AVM/ModuleProposal).
```
Comment (eriqua) [inline README.md:5]: We could add [GH markdown alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) to highlight important info, as recently done for automatic replies. For example

```suggestion
Going forward, new modules need to be developed and published in accordance with the [AVM specifications](https://azure.github.io/Azure-Verified-Modules/specs/module-specs/). 
> [!CAUTION]
> Module proposals for new, non-AVM modules are no longer accepted. 
To propose a new AVM module, you can file a Module Proposal, [here](https://aka.ms/AVM/ModuleProposal).
```

Comment (matebarabas) [reply README.md:5]: I didn't want to make the landing page overly colorful 😊, but as long as it's only one callout, I think we're good. Following the official definitions for the various levels of call-outs however, I'd go with `IMPORTANT` and not `CAUTION`.
image

Comment (eriqua) [reply README.md]: Sounds fair, it was only an example, not an actual suggestion. You can resolve the comment.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
+This repo contains the source code of all currently available Bicep modules in the Bicep public module registry (i.e., all AVM Bicep modules).
 
 ## Available Modules
 
-To view available modules and their versions, go to Bicep Registry Module Index ([https://aka.ms/br-module-index](https://aka.ms/br-module-index)).
+To view all available AVM Bicep modules, go to AVM Bicep Module Index ([https://aka.ms/AVM/ModuleIndex/Bicep](https://aka.ms/AVM/ModuleIndex/Bicep)).
 
 ## Contributing
 
-We only accept contributions from Microsoft employees at this time. Teams within Microsoft can refer to [Contributing to Bicep public registry](./CONTRIBUTING.md) for information on contributing modules. External customers can propose new modules or report bugs by opening an [issue](https://github.com/Azure/bicep-registry-modules/issues).
```
Comment (eriqua) [inline README.md:23]: What about previously linked pages, such as [./CONTRIBUTING.md](./CONTRIBUTING.md) should we retire it as well? 
I'd suggest at least to add a notice at the top mentioning the content is outdated and will be removed next.

**Review by shenglol — APPROVED** [review]
:shipit:

File: README.md (hunk 2 above)
Comment (matebarabas) [reply README.md:23]: Here's the new PR, taking care of this: https://github.com/Azure/bicep-registry-modules/pull/1448

**Pull request merged**